templates/publisher/publicise.html

```diff
@@ -0,0 +1,516 @@
+license terms or to stop distributing the Work at any time; provided, however
+that any such election will not serve to withdraw this License (or any other
+license that has been, or is required to be, granted under the terms of this
+License), and this License will continue in full force and effect unless terminated
+as stated above.
+
+   8. Miscellaneous
+
+a. Each time You distribute or publicly digitally perform the Work, the Licensor
+offers to the recipient a license to the Work on the same terms and conditions
+as the license granted to You under this License.
+
+b. If any provision of this License is invalid or unenforceable under applicable
+law, it shall not affect the validity or enforceability of the remainder of
+the terms of this License, and without further action by the parties to this
+agreement, such provision shall be reformed to the minimum extent necessary
+to make such provision valid and enforceable.
+
+c. No term or provision of this License shall be deemed waived and no breach
+consented to unless such waiver or consent shall be in writing and signed
+by the party to be charged with such waiver or consent.
+
+d. This License constitutes the entire agreement between the parties with
+respect to the Work licensed here. There are no understandings, agreements
+or representations with respect to the Work not specified here. Licensor shall
+not be bound by any additional provisions that may appear in any communication
+from You. This License may not be modified without the mutual written agreement
+of the Licensor and You.
+
+Creative Commons is not a party to this License, and makes no warranty whatsoever
+in connection with the Work. Creative Commons will not be liable to You or
+any party on any legal theory for any damages whatsoever, including without
+limitation any general, special, incidental or consequential damages arising
+in connection to this license. Notwithstanding the foregoing two (2) sentences,
+if Creative Commons has expressly identified itself as the Licensor hereunder,
+it shall have all rights and obligations of Licensor.
+
+Except for the limited purpose of indicating to the public that the Work is
+licensed under the CCPL, neither party will use the trademark "Creative Commons"
+or any related trademark or logo of Creative Commons without the prior written
+consent of Creative Commons. Any permitted use will be in compliance with
+Creative Commons' then-current trademark usage guidelines, as may be published
+on its website or otherwise made available upon request from time to time.
+
+Creative Commons may be contacted at http s ://creativecommons.org/.</code></pre>
+          </div>
+        </div>
+      </div>
+
+      <div class="vbox-close">X</div>
+    </div>
+  </div>
+
+</div>
+{% endblock %}
+
+{% block scripts %}
+<script>
+  Raven.context(function () {
+    var languagePicker = document.querySelector('.js-language-select');
```

Comment:
Would be great if we could limit the amount of inline JS. It's not linted, prettified, minified, cached.